rllib/algorithms/algorithm_config.py

```diff
@@ -552,6 +554,8 @@ def rollouts(
         synchronize_filter: Optional[bool] = None,
         compress_observations: Optional[bool] = None,
         enable_tf1_exec_eagerly: Optional[bool] = None,
+        sampler_perf_stats_use_ema: Optional[bool] = False,
```

Comment:
Defaults should be `None` here. Otherwise, the method would not now whether to override these (if not None) or just leave 'em as-is (if None).

Reply:
oh nice catch.